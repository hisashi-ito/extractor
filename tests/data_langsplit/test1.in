df6fa1abb58549287111ba8d776733e9 uri:http://017ssmys.measychina.cn/za/11hbg/
海报psd素材_四驱车 田宫 绝版_植物百科通
新闻资讯
植物科学
家庭养花
园林养护
植物图片
df6fa1abb58549287111ba8d776733e9 uri:http://02repair.blog.fc2.com/blog-entry-192.html
コラージュ・イン・ア・ボトル スタカニ用ATC作成中☆
コラージュ・イン・ア・ボトル
collages in a bottle
スポンサーサイト
上記の広告は１ヶ月以上更新のないブログに表示されています。新しい記事を書く事で広告が消せます。	-------- : スポンサー広告 : このページのトップへ
スタカニ用ATC作成中☆