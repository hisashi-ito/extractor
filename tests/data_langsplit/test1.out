df6fa1abb58549287111ba8d776733e9 uri:http://917ssmys.measychina.cn/za/11hbg/ language:zh offset:9 bytes:112
素材_四驱车 田宫 绝版_植物百科通
新闻资讯
植物科学
家庭养花
园林养护
植物图片

df6fa1abb58549287111ba8d776733e9 uri:http://02repair.blog.fc2.com/blog-entry-192.html language:ja offset:0 bytes:58
コラージュ・イン・ア・ボトル スタカニ用
df6fa1abb58549287111ba8d776733e9 uri:http://02repair.blog.fc2.com/blog-entry-192.html language:ja offset:61 bytes:56
作成中☆
コラージュ・イン・ア・ボトル

df6fa1abb58549287111ba8d776733e9 uri:http://02repair.blog.fc2.com/blog-entry-192.html language:en offset:117 bytes:21
collages in a bottle

df6fa1abb58549287111ba8d776733e9 uri:http://02repair.blog.fc2.com/blog-entry-192.html language:ja offset:138 bytes:248
スポンサーサイト
上記の広告は１ヶ月以上更新のないブログに表示されています。新しい記事を書く事で広告が消せます。	-------- : スポンサー広告 : このページのトップへ
スタカニ用
